docs/DeveloperGuide.adoc

```diff
@@ -230,6 +230,119 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
+
+To add the comment of a client, the cm/ tag is used with either the add or edit commands, followed by the comment.
+
+An example of the add command would be: add n/David ... cm/interested in football
+
+An example of the edit command would be: edit 1 cm/interested in bascketball
+
+==== Design Considerations
+
+===== Aspect: Implementation of comment
+
+* **Alternative 1 (current choice):** User must add comment when adding client.
+** Pros: Information consistency.
+** Cons: User may not always want to add comment to clients.
+* **Alternative 2:** User can optionally add the comments in a separate command.
+** Pros: Will use less memory (e.g. number of comments are less than or equal to number of users).
+** Cons: May be more difficult to manage contacts without some comments.
+//end::AddComment[]
+
+//tag::archive[]
+=== Archive feature
+==== Current Implementation
+The archive feature allows the salesperson to archive clients that he no longer contacts, but does not want to delete as he possible might want to contact him/her again. Once archived, the client is not shown in the list. To display the client, either find or listall command has to be used. The client is then displayed in a red font with strikethrough, indicating that he/she is archived.
+
+The archival feature is implemented by adding a boolean variable isArchived in the Person model. Listing is filtered through the use of a predicate, PREDICATE_SHOW_UNARCHIVED_PERSONS, that will filter for those clients that are unarchived
+
+==== Design Considerations
+
+===== Aspect: find shows archived clients
+
+* **Alternative 1 (current choice):** Find shows archived clients.
+** Pros: Allows the user to quickly find specific clients that he wants to unarchive.
+** Cons: Goes against the purpose of not cluttering up the UI with archived clients.
+* **Alternative 2:** Find only displays unarchived clients.
+** Pros: Will not clutter up the UI with archived clients so that the user can quickly find clients that are unarchived.
+** Cons: It is hard to find a specific archived person. User has to scroll through listall.
+
+===== Aspect: isArchived is stored in a boolean in the Person model
+
+* **Alternative 1 (current choice):** Store whether a client is archived in the Person model as a boolean
+** Pros: Simple to implement
+** Cons: Uses more space than the alternative.
+* **Alternative 2:** Keep two separate lists of persons, archived and unarchived.
+** Pros: Uses less space than the alternative.
+** Cons: Harder to implement, more changes have to be made to existing codebase.
+// end::archive[]
+
+//tag::timezone[]
+=== TimeZone feature
+==== Current Implementation
+The time zone feature allows the sales person to add a time zone for their clients. This will allow for simplified appointment scheduling between the sales person and their international clients.
+
+To add the time zone of a client, the tz/ tag is used with either the add or edit commands, followed by the time zone.
```

Comment:
I suspect you are abusing the word "tag" to talk about many fields. Tag itself is a field. Please fix this writing error!